tests/src/behaviors/tx-per-second.ts

```diff
@@ -0,0 +1,186 @@
+import { Player } from '@holochain/tryorama'
+import * as _ from 'lodash'
+import { v4 as uuidv4 } from "uuid";
+import { batcher as batchOfConfigs } from './config'
+
+const delay = ms => new Promise(r => setTimeout(r, ms))
+
+type Players = Array<Player>
+
+export const defaultConfig = {
+    nodes: 1, // Number of machines
+    conductors: 1, // Conductors per machine
+    instances: 2, // Instances per conductor
+    endpoints: null, // Array of endpoints for Trycp
+}
+
+function prettyCellID(id) {
+    return JSON.stringify(id[1].hash)
```

Comment:
`CellID.hash.toString('base64')` can display `Buffer` in a human readable format. Can be helpful in the future.